modules/web-console-configuration.adoc

```diff
@@ -29,10 +28,5 @@ spec:
 status:
   consoleURL: "" <2>
 ----
-<1> Specify the URL of the page to load when a user logs out of the web console.
-If you do not specify a value, the user returns to the login page for the
-web console. Specifying a `logoutRedirect` URL allows your users to perform
-single logout (SLO) through the identity provider to destroy their single
-sign-on session.
-<2> The web console URL. You cannot modify this parameter value. If you do, the
-cluster resets it to the default value.
+<1> Specify the URL of the page to load when a user logs out of the web console. If you do not specify a value, the user returns to the login page for the web console. Specifying a `logoutRedirect` URL allows your users to perform single logout (SLO) through the identity provider to destroy their single sign-on session.
+<2> The web console URL. To update this to a custom value, see Customizing the web console in {product-title}.
```

Comment:
The title here doesn't match the module title. I also think that this needs to be in bold, but I can't find a ruling in our doc guidelines.